core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java

```diff
@@ -154,6 +154,13 @@ public void post(Object entity, InputStream stdin, ResultCallback<Frame> resultC
         if (stdin != null) {
             // FIXME there must be a better way of handling it
             okHttpClient = okHttpClient.newBuilder()
+                .hostnameVerifier(new HostnameVerifier() {
```

Comment:
doesn't `okHttpClient.newBuilder` inherit parent's `hostnameVerifier`?

Reply:
The default implementation is OkHostnameVerifier. With it, you can't avoid the hostname check.

Reply:
`okHttpClient.newBuilder` creates a builder from an existing instance. If your existing instance has `hostnameVerifier` already set, doesn't it get inherited?

Reply:
Honesty, I don't test it. I will, thank you for the idea.

Reply:
Tested, don't work :/
OkHttp client is initialized in OkHttpDockerCmdExecFactory.init() method.